src/main/java/com/linkedin/kmf/services/TopicManagementServiceManager.java

```diff
@@ -0,0 +1,154 @@
+/**
+ * Copyright 2016 LinkedIn Corp. Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+
+package com.linkedin.kmf.services;
+
+import com.linkedin.kmf.apps.configs.MirrorPipelineMonitorConfig;
+import com.linkedin.kmf.services.configs.CommonServiceConfig;
+import com.linkedin.kmf.services.configs.ConsumeServiceConfig;
+import com.linkedin.kmf.services.configs.TopicManagementServiceConfig;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ThreadFactory;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class TopicManagementServiceManager implements Service {
+  private static final Logger LOG = LoggerFactory.getLogger(TopicManagementServiceManager.class);
+
+  private double _partitionsToBrokerRatio;
+  private String _name;
+  private List<String> _zookeeperUrls;
+  private AtomicBoolean _running;
+  private final ScheduledExecutorService _managementExecutor;
+  private Map<String, TopicManagementService> _services;
+
+  public TopicManagementServiceManager(Map<String, Object> props, List<String> zookeeperUrls, String topic,
```

Comment:
If we want to add a new service that manages topic across multiple clusters, can this service has its own config and takes a properties map as parameter of its constructor, just like every other services? It probably needs its own config class.